clang/lib/Basic/Targets/SPIR.h

```diff
@@ -63,7 +63,7 @@ class LLVM_LIBRARY_VISIBILITY SPIRTargetInfo : public TargetInfo {
     VLASupported = false;
     LongWidth = LongAlign = 64;
     if (Triple.getEnvironment() == llvm::Triple::SYCLDevice &&
-        getenv("ENABLE_INFER_AS")) {
+        !getenv("DISABLE_INFER_AS")) {
```

Comment:
Even if it is a temporary solution, naming the environment variable such as `SYCL_CLANG_DISABLE_INFER_AS` or whatever so we can understand among the crowded output of a `printenv`.

Reply:
I'll change the variable name in a follow-up commit.